contracts/base/LendingPool.sol

```diff
@@ -231,11 +271,6 @@ contract LendingPool is Base, LendingPoolInterface {
         require(transferFromResult, "TransferFrom wasn't successful.");
     }
 
-    function requireEnoughLendingTokenBalance(uint256 amount) private view {
```

Comment:
Why is this being taken out?  Is it because all the funds are all in Compound, and a withdrawal from Compound that's "too much" will simply fail?  We may want a better way of handling this in the future (e.g. failure with a reason that's communicated to the user), depending on what the final withdrawal mechanism will be.

Reply:
yes I could potentially add it back in. I try to prioritise reducing the number of unnecessary checks, as users don't like to use apps that have high gas costs. But if you'd like this back in because of the error message then I can do so